ui/containers/WorkspaceDetails/sagas.ts

```diff
@@ -391,7 +418,7 @@ function* changeWorkspaceOwnerRequested() {
     yield call(Api.changeWorkspaceOwner, token, id, ownerDn);
   } catch (err) {
     yield put(
-      setNotificationStatus(NotificationType.Error, `Failed to change the owner of ${name}: ${err.toString()}`)
+      setNotificationStatus(NotificationType.Error, `Failed to change the owner of ${name}: ${err.toString()}`, 0)
```

Comment:
Maybe use a default argument instead of updating all of these? I can go either way.

```
export const setNotificationStatus = (type: NotificationType, message: string, duration: number = 0)
```

Reply:
@afoerster , I agree with your opinion. @shafi1379 we can use `0` as default value.

Reply:
@afoerster @rk-fsdev I agreed, I will make changes